src/main/java/org/opensearch/ad/transport/AnomalyResultTransportAction.java

```diff
@@ -736,6 +754,12 @@ private void handleRCFResults(int numFeatures) {
                     listener.onFailure(new InternalFailure(adID, NO_MODEL_ERR_MSG));
                     return;
                 }
+                try {
+                    long totalUpdates = rcfResults.get(0).getTotalUpdates();
+                    adTaskManager.updateLatestRealtimeTask(detector.getDetectorId(), totalUpdates, intervalMinutes, null);
```

Comment:
This can clear the detector's error, right? Current implementation would set error after observing rcf and threshold model and cold start error and create a final error in job scheduler runner.

Reply:
Yeah, agree. The first commit is some testing code to verify if can get total updates and update realtime init progress successfully. The second commit should fix it.